README.md

```diff
@@ -45,4 +43,12 @@ Now when you go back to the UI in your browser it should look like this and it c
 ## Experiment!
 Now that everything is running the Gaia [rules](./src/access_control.ruleset) can be modified and extended to change behaviors. We encourage you to experiment to see how changes affect behavior and to imagine how Gaia could be used for other project ideas you may have.
 
-If you have any questions or feedback please reach out to us at Gaia at info@gaiaplatform.io or directly to the author of this example at steve@gaiaplatform.io
+If you changed the ruleset or the C++ code, rebuild it.
+```
+cd build
+make -j$(nproc)
+```
+
+If you changed the schema DDL, run `./rebuild.sh`. This will erase the data in the database, but it's necessary to sync the schema with the state of the database.
+
+If you have any questions or feedback please reach out to us at Gaia at [info@gaiaplatform.io](mailto:info@gaiaplatform.io) or directly to the author of this example at [steve@gaiaplatform.io](mailto:steve@gaiaplatform.io).
```

Comment:
Let's go ahead and take my name out. I think info@gaiaplatform.io *should* be sufficient.

Reply:
"Cleaning up" the ruleset should happen after we've sandboxified the app. Waiting for more eyes would be a speed bump and reduce the time that Kenneth could help out.

I will remove your email, good idea.